sechub-cli/src/mercedes-benz.com/sechub/cli/tarbuilder_test.go

```diff
@@ -0,0 +1,151 @@
+	sechubTestUtil.CreateTestFile(filepath2, 0644, content, t)
+	sechubTestUtil.CreateTestFile(filepath3, 0644, content, t)
+	sechubTestUtil.CreateTestFile(filepath4, 0644, content, t)
+	sechubTestUtil.CreateTestFile(filepath5, 0644, content, t)
+
+	// create configs
+	namedBinariesScanConfig1 := NamedBinariesScanConfig{
+		Name:       "testname1",
+		FileSystem: FileSystemConfig{Files: []string{filepath4, filepath5}, Folders: []string{dirname1}},
+	}
+
+	namedBinariesScanConfig2 := NamedBinariesScanConfig{
+		Name:       "testname2",
+		FileSystem: FileSystemConfig{Folders: []string{dirname2}},
+	}
+
+	context.sechubConfig.Data.Binaries = []NamedBinariesScanConfig{namedBinariesScanConfig1, namedBinariesScanConfig2}
+	context.binariesTarFileName = dir + "/test.tar"
+
+	/* execute */
+	createBinariesTarFile(&context)
+
+	/* test */
+	list, _ := sechubUtil.ListContentOfTarFile(context.binariesTarFileName)
+	sechubTestUtil.AssertSize(list, 5, t)
+	sechubTestUtil.AssertContains(list, archiveDataPrefix+"/"+namedBinariesScanConfig1.Name+filepath1, t)
+	sechubTestUtil.AssertContains(list, archiveDataPrefix+"/"+namedBinariesScanConfig1.Name+filepath4, t)
+	sechubTestUtil.AssertContains(list, archiveDataPrefix+"/"+namedBinariesScanConfig1.Name+filepath5, t)
+	sechubTestUtil.AssertContains(list, archiveDataPrefix+"/"+namedBinariesScanConfig2.Name+filepath2, t)
+	sechubTestUtil.AssertContains(list, archiveDataPrefix+"/"+namedBinariesScanConfig2.Name+filepath3, t)
+}
+
+func Test_createBinariesTarFile_DataBinariesSectionWorksWithRelativePathes(t *testing.T) {
+	/* prepare */
+	var context Context
+	var config Config
+	var sechubConfig SecHubConfig
+	context.config = &config
+	context.sechubConfig = &sechubConfig
+
+	// dir is relative path
+	dir := "sechub-cli-tmptest"
+	sechubTestUtil.CreateTestDirectory(dir, 0755, t)
+	defer os.RemoveAll(dir)
+
+	// create subdirectories
+	dirname1 := dir + "/sub1"
+	sechubTestUtil.CreateTestDirectory(dirname1, 0755, t)
+
+	// create files
+	filepath1 := dirname1 + "/file-sub1.bin"
+	filepath2 := dir + "/standalone1.bin"
+	content := []byte("Binary :-)\n")
+	sechubTestUtil.CreateTestFile(filepath1, 0644, content, t)
+	sechubTestUtil.CreateTestFile(filepath2, 0644, content, t)
+
+	// create configs
+	namedBinariesScanConfig1 := NamedBinariesScanConfig{
+		Name:       "testname1",
+		FileSystem: FileSystemConfig{Files: []string{filepath2}, Folders: []string{dirname1}},
+	}
+
+	context.sechubConfig.Data.Binaries = []NamedBinariesScanConfig{namedBinariesScanConfig1}
+	context.binariesTarFileName = dir + "/test.tar"
+
+	/* execute */
+	createBinariesTarFile(&context)
+
+	/* test */
+	list, _ := sechubUtil.ListContentOfTarFile(context.binariesTarFileName)
+	sechubTestUtil.AssertContains(list, archiveDataPrefix+"/"+namedBinariesScanConfig1.Name+"/"+filepath1, t)
+	sechubTestUtil.AssertContains(list, archiveDataPrefix+"/"+namedBinariesScanConfig1.Name+"/"+filepath2, t)
+	sechubTestUtil.AssertSize(list, 2, t)
```

Comment:
Please move this line up before the `AssertContains` statements.

Reply:
line has been moved